### PR TITLE
perl 5.30.1

### PR DESCRIPTION
**Pull request by dawidd6**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Several revision bumps needed:

```
brew-test-bot.mojave.linkage --test subversion
brew-test-bot.mojave.linkage --test vim
brew-test-bot.mojave.linkage --test vim@7.4
brew-test-bot.mojave.linkage --test weechat
```

**Comment by dawidd6** [comment]
```
Error Message
failed: brew linkage --test weechat
Stacktrace
        Missing libraries:
  /usr/lib/libruby.2.6.dylib
```

**Comment by Bo98** [comment]
Gonna wait on this until after Python 3.8 (#45337) due to the potential of merge conflicts.

**Comment by Bo98** [comment]
This fixed weechat on Mojave for me:

```ruby
if MacOS.version == :mojave && MacOS::CLT.installed?
  ENV["SDKROOT"] = ENV["HOMEBREW_SDKROOT"] = MacOS::CLT.sdk_path(MacOS.version)
end
```

**Comment by fxcoudert** [comment]
See the fix committed. And please rebase on top of master.

**Comment by dawidd6** [comment]
```
Error: 1 problem in 1 formula detected
perl:
  * HEAD: The URL https://perl5.git.perl.org/perl.git is not a valid git URL
```

File: Formula/perl.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Perl < Formula
   desc "Highly capable, feature-rich programming language"
   homepage "https://www.perl.org/"
-  url "https://www.cpan.org/src/5.0/perl-5.30.0.tar.gz"
-  sha256 "851213c754d98ccff042caa40ba7a796b2cee88c5325f121be5cbb61bbf975f2"
+  url "https://www.cpan.org/src/5.0/perl-5.30.1.tar.gz"
+  sha256 "bf3d25571ff1ee94186177c2cdef87867fd6a14aa5a84f0b1fb7bf798f42f964"
   head "https://perl5.git.perl.org/perl.git", :branch => "blead"
```
Comment (fxcoudert) [inline Formula/perl.rb]:
```suggestion
  head "https://github.com/perl/perl5.git", :branch => "blead"
```

**Comment by fxcoudert** [comment]
perl `head` needs to be updated, and you should squash the two `weechat` commits together. Then I think it's good :)

**Pull request closed without merging**